changelog/59865.fixed

```diff
@@ -0,0 +1,3 @@
+Update the Salt installer to use Python 3.8 for its python environment
+Update libsodium.dll to 1.0.18
+Update OpenSSL dlls to 1.1.1k
```

Comment:
**question**: Does` towncrier handle this sort of entry properly? If not, I'd suggest something like:

> Update Windows build deps & DLLs, Use Python 3.8, libsodium.dll 1.0.18, OpenSSL dlls to 1.1.1k

(that should be `1.1.1k`?)